indico/modules/events/contributions/client/js/index.jsx

```diff
@@ -29,30 +30,42 @@ import PublicationSwitch from './PublicationSwitch';
     if (!editableSubmissionButton) {
       return;
     }
-
     const availableTypes = JSON.parse(editableSubmissionButton.dataset.availableTypes);
     const {eventId, contributionId, contributionCode} = editableSubmissionButton.dataset;
 
-    let responses;
+    let responses, lastRevFiles;
     try {
-      responses = await Promise.all(
-        availableTypes.map(type => indicoAxios.get(fileTypesURL({confId: eventId, type})))
-      );
+      responses = await Promise.all([
+        Promise.all(
+          availableTypes.map(type => indicoAxios.get(fileTypesURL({confId: eventId, type})))
+        ),
+        indicoAxios
+          .get(paperInfoURL({confId: eventId, contrib_id: contributionId}))
+          // eslint-disable-next-line no-unused-vars
+          .catch(e => undefined),
```

Comment:
`() => undefined` to avoid the unused var

But I'd rather override `validateStatus` in the options for this particular request to avoid rejecting the promise on the expected error status code altogether...

Reply:
Didn't know about `validateStatus`. I tried searching for something that would catch them.
For the sake of simplicity, I made it resolve errors between 200 and 404, since I don't think resolving errors between 400 and 404 is a big problem.

Reply:
I don't think that's a good idea. Especially 403 is something where we usually want error handling, since a Forbidden error is rather unexpected there...

Reply:
I didn't expect that to happen in the front-end but I've made it strict.